File: app/src/main/java/com/electrosim/LinkPointInterface.java
package com.electrosim;

import android.graphics.Canvas;
import android.view.MotionEvent;

public interface LinkPointInterface {


	public boolean pointHitTest(MotionEvent event);
	public void setConnector(ConnectorInterface connector);
	
	public float getY();
	
	public void draw(Canvas canvas);
	
	public float getX();
}

File: app/src/main/java/com/electrosim/LinkPointInterface.kt
package com.electrosim

import android.graphics.Canvas
import android.view.MotionEvent

interface LinkPointInterface {
    fun pointHitTest(event: MotionEvent): Boolean
    fun setConnector(connector: ConnectorInterface)
    fun getY(): Float
    fun draw(canvas: Canvas)
    fun getX(): Float
} 